statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -913,24 +914,30 @@ def select_order(self, maxlags=None, trend="c"):
 
         return LagOrderResults(ics, selected_orders)
 
+    @property
+    def exog_names(self):
```

Comment:
why did you turn this into a property?

Reply:
Better docstring and to match endog_names